apps/vr-tests/src/stories/Callout.stories.tsx

```diff
@@ -0,0 +1,37 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import * as React from 'react';
+import Screener, { Steps } from 'screener-storybook/src/screener';
+import { storiesOf } from '@storybook/react';
+import { FabricDecoratorTall } from '../utilities';
+import { Callout, Link, DirectionalHint } from 'office-ui-fabric-react';
+
+let targetElement = document.getElementById('button') as HTMLElement;
+
+storiesOf('Callout', module)
+  .addDecorator(FabricDecoratorTall)
+  .addDecorator(story => (
+    // tslint:disable-next-line:jsx-ban-props
+    <div>
+      <button id='button'>Hey</button>
+      { story() }
+    </div>
+  ))
+  .addDecorator(story => (
+    <Screener
+      steps={ new Screener.Steps()
+        .snapshot('default', { cropTo: '.ms-Layer' })
```

Comment:
My intent here was to crop to `.ms-Layer` because the Callout doesn't 'fit' into `.test-wrapper`. I've experimented with toggling `doNotLayer` and changing `gapspace` in order to get a snapshot of the full Callout with beak. I thought this solution would work but I just realized I'm getting different behavior on this when I manually refresh the Storybook vs when Storybook auto refreshes when saving an edit.